src/fiskaltrust.Launcher/ServiceInstallation/LinuxSystemD.cs

```diff
@@ -7,10 +7,14 @@ public class LinuxSystemD : ServiceInstaller
     {
         private static readonly string _servicePath = "/etc/systemd/system/";
         private readonly string _serviceName = "fiskaltrustLauncher";
-
-        public LinuxSystemD(string? serviceName, LauncherExecutablePath launcherExecutablePath) : base(launcherExecutablePath)
+        private readonly string _serviceUser;
+        private readonly string _requiredDirectory = "/var/lib/fiskaltrust";
```

Comment:
the path to this directory is actually set in the `LauncherConfiguration.ServiceDirectory` we should get it from the launcher configuration